frontend/src/core/notification/components/NotificationPanel.css

```diff
@@ -0,0 +1,32 @@
+.notification-panel {
+    background: rgba(51, 57, 65, 0.9);
+    color: #7BC876;
+    cursor: pointer;
+    font-style: italic;
+    left: 0;
+    margin: 0;
+    position: fixed;
+    top: -60px;
+    width: 100%;
+    z-index: 100;
+    transition: 500ms;
```

Comment:
I'd reduce that to `200ms`, to keep the animation as "fast" as on `Translation.Current`.